src/main/java/com/parkhomovsky/bookstore/exception/GlobalExceptionHandler.java

```diff
@@ -46,15 +46,18 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
     }
 
     @ExceptionHandler(DataIntegrityViolationException.class)
-    public ResponseEntity<Object> handleUniqueDataDuplicate(
-            DataIntegrityViolationException ex
-    ) {
+    public ResponseEntity<Object> handleUniqueDataDuplicate(DataIntegrityViolationException ex) {
         Throwable rootCause = ExceptionUtils.getRootCause(ex);
         String errorMessage = "An error occurred while processing the request.";
         if (rootCause != null) {
             errorMessage = "Error: " + rootCause.getMessage();
         }
-        return new ResponseEntity<>(errorMessage, HttpStatus.CONFLICT);
+        Map<String, Object> errorResponse = new LinkedHashMap<>();
+        errorResponse.put(TIMESTAMP, LocalDateTime.now());
```

Comment:
It is better to create object something like ErrorResponse/ErrorResponseDto, instead of using map